src/query_ctx.h

```diff
@@ -102,7 +113,7 @@ bool QueryCtx_LockForCommit(void);
  * 2. Unlock graph R/W lock
  * 3. Close key
  * 4. Unlock GIL */
-void QueryCtx_UnlockCommit(void);
+void QueryCtx_UnlockCommit(OpBase *writer_op);
```

Comment:
The comments describing `QueryCtx_LockForCommit` and `QueryCtx_UnlockCommit` should be updated to reflect the new changes.

Reply:
done